bioinflib/bioinflib_folder_1/filter_module.py

```diff
@@ -0,0 +1,11 @@
+def find_gc_content(seq: str) -> float:
+    content = 0
+    for nucl in seq:
+        if nucl == 'G' or nucl == 'g' or nucl == 'C' or nucl == 'c':
```

Comment:
По заданию могли встретиться строчные буквы? Если там только прописные, то можно убрать проверку на строчные
```suggestion
        if nucl == 'G' or nucl == 'C':
```

Reply:
Ну я думаю лучше в общем случае работать, да